src/main.cpp

```diff
@@ -333,6 +363,53 @@ class multi_visitor {
 QLJS_STATIC_ASSERT_IS_PARSE_VISITOR(
     multi_visitor<debug_visitor, debug_visitor>);
 
+void handle_options(quick_lint_js::options o) {
```

Comment:
Nit: To keep the diff readable, move this code back to where it was originally. (Introduce an unnamed namespace block as needed.)